### DISCUSSION TITLE
WhatsApp quick response

### DISCUSSION THREAD

**Discussion by giampaulo** [zepp-health/zeppos-forum#107] [Developer Feedback]
Hi all. 
I was thinking if is possible develop a mini app to gts 4 mini to answer some messages from WhatsApp with quick responses, like example thumbs up. Using the notification to select the message to answer. 

Do you guys have ideia if this is possible with os 1.0?

**Comment by vaas1993** [comment]
Android system supports this feature. Of course, this requires additional adaptation work by ZeppOS and applet developers. At present, ZeppOS 2.0 does not seem to support this feature

ps: I also want to have a small program that can support WeChat quick reply

**Comment by giampaulo** [comment]
Thanks for replying. Ok. I got it. Have some kind of chance to works in IOS system ?


Att

Giampaulo Hanauer

***@***.***
Em 10 de mar. de 2023 03:23 -0300, vshen ***@***.***> escreveu:

**Comment by Cotrox** [comment]
It will be amazing if you find a way to do it for iOS. I suggested to developers to look at what is doing Microsoft with the App Phone Link that finally let iPhone users to answer, read and do different things also on iOS. There are a lot of limitations obviously but they are working well on them to find a way!

**Reply by fwz233** [reply]
我也这么觉得
